### DISCUSSION TITLE
How can I ask the user for a location?

### DISCUSSION THREAD

**Discussion by diegoqueres** [rubenlagus/TelegramBots#1334] [Q&A]
When interacting with the user, the bot must ask several questions and in one of them it will ask the user  to share the location of an address. It won't necessarily be his current location, he could provide another location.
I thought about sending a simple message, waiting for the user's response and somehow interpreting whether the response contains location data. Which would wait for the user's response where they would share the location that the bot is waiting for.
I'm using Ability Extension and ReplyFlow to interact with the user.
Is there any better way to do this?

**Comment by rubenlagus** [comment]
@diegoqueres  You can use the location sharing feature that Telegram provide. It allows users to send any location.
